src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -431,18 +424,104 @@ public static void setLocalIndexes(Collection<ComputedContingencyElement> elemen
 
     }
 
+    static class LfContingency {
+        private final Contingency contingency;
+        private final Collection<ComputedContingencyElement> elements;
+
+        public LfContingency(Contingency contingency, Map<ContingencyElement, Collection<ComputedContingencyElement>> defaultPropagation) {
```

Comment:
`defaultPropagation` is not well chosen. Maybe you should introduce something close to `LfBranchTripping` world.